playbooks/common/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: ../openshift-cluster/evaluate_groups.yml
+
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: List known health checks
+    action: openshift_health_check
+    when: openshift_checks is undefined or not openshift_checks
```

Comment:
This is good for the two cases in the condition, but doesn't cover other cases such as when there is a typo in a check name.

I was thinking we could have a "preflight check" for the `openshift_health_check` plugin itself, implemented in the plugin perhaps with a flag like `dry-run: true`. But that doesn't have to be part of this PR.